### DISCUSSION TITLE
Release v0.13.3-beta

### DISCUSSION THREAD

**Discussion by saubyk** [Ride-The-Lightning/RTL#1158] [Announcements]
Change highlights for this minor release:
- Angular update to version 15, minor impact on the look and feel of the application
- Eclair (bug fixes)
    - Breaking changes due to updates to the Eclair APIs with v0.8.0
    - Sorting reset bug on performing actions on grids
    - Real time update bug for forwarding history
- Fixed inconsistent color code for inactive channels on the channels page and the dashboard

-----------------------------------------------------------------------------------------------

<hr /><em>This discussion was created from the release <a href='https://github.com/Ride-The-Lightning/RTL/releases/tag/v0.13.3'>Release v0.13.3-beta</a>.</em>

**Comment by rmd84** [comment]
I am unable to open a channel. FIlling out the form and hitting open channel does nothing. I am using the myNode implementation

<img width="722" alt="image" src="https://user-images.githubusercontent.com/10411960/222921151-e00717f1-bc65-4982-82ec-ed40a42b441d.png">

**Comment by saubyk** [comment]
Hi @rmd84 what's the RTL version that you're running?

**Reply by saubyk** [reply]
I would recommend upgrading to the latest version (0.13.6), if you haven't already

**Reply by rmd84** [reply]
Hi @saubyk I am on 0.13.6 Beta, has it exited beta?

**Reply by saubyk** [reply]
@ShahanaFarooqui we need to verify if there is a change in LND's Api response for open channel, which is resulting in an error condition not being caught on the UI

**Comment by sat-hoarder** [comment]
I really appreciate the work you guys put into this, but the new theme/colors make it almost unusable in darkmode.  Anyway to customize the style?

**Reply by ShahanaFarooqui** [reply]
Did you try Node Config -> Node Settings tab -> Customization section. We have 5 colors & 2 modes to choose from.

**Reply by sat-hoarder** [reply]
Yes, it just seems like the all the font's got bigger and the colors are lighter in shade.  I know this is really petty stuff so sorry about that.  I've just switched over to light mode and I'll get used to it.  I appreciate the response!

**Reply by ShahanaFarooqui** [reply]
With Angular material upgrade, the theme changed and fonts got bigger but colors are still same. But I will check if I can darken them a little for dark mode in the next release. Could you please open an issue for this?

**Reply by ShahanaFarooqui** [reply]
BTW this stuff is not petty. It is very important to feel comfortable with the theme while using the application. I am just not that good with UX, so I wasn’t able to see the difference.
